projects/lib/src/dynamic-views/item.component.ts

```diff
@@ -15,10 +15,10 @@ import { Subscription } from 'rxjs';
 import { ConditionsService } from '../conditions/conditions.service';
 import { ComponentConfig, ConfigService } from '../configuration/config.service';
 import { TemplateNameDirective } from '../utils';
-import { DragDropService } from './drag-drop.service';
+import { ContainerIndex, DragDropService } from './drag-drop.service';
 
 @Component({
-  selector: '[uib-item]',
+  selector: 'uib-item, [uib-item]',
```

Comment:
why use `<uib-item>` instead of `<div [uib-item=...>` ?
Using div is more safe because this is what is exported by the schematics, so we are sure it's the same HTML structure before and after the export.

Reply:
used on my previous works, to distinguish the template injection workflow. The actual code works without it now. I can safely remove it